roles/ceph-common/tasks/main.yml

```diff
@@ -64,24 +64,16 @@
   tags:
     - package-install
 
-- name: get ceph rhcs version
-  shell: rpm -q --qf "%{version}\n" ceph-common | cut -f1,2 -d '.'
+- name: get ceph version
+  shell: ceph --version | cut -f3 -d ' '
```

Comment:
Sounds good.  Or better yet, how about `set -o pipefail; ceph --version | cut -f3 -d ' '`.  This will return a fail if any command in the pipeline fails, and is a little cleaner ;-).  I found it in this [thread](http://stackoverflow.com/questions/1221833/bash-pipe-output-and-capture-exit-status)